backends/qualcomm/partition/common_defs.py

```diff
@@ -11,8 +11,9 @@
 not_supported_operator = [
     exir_ops.edge.aten.arange.start_step,
     exir_ops.edge.aten.clone.default,
-    exir_ops.edge.aten.index.Tensor,
     exir_ops.edge.aten.full.default,
+    exir_ops.edge.aten.index.Tensor,
+    exir_ops.edge.aten.index_put.default,
```

Comment:
Maybe these two lines can be landed seperately? It's not supported anyway and we can decouple it from this large PR

Reply:
I think we still need to have it. We need `aten.index.Tensor` for slicing kv_cache and feed to individual LlamaAttention layer. `aten.index_put` is used to update kv_mask after finishing each inference.
These two operators are not supported yet by qualcomm backend, we'll need partitioner to identify them and make them fallback to CPU.

Reply:
I see, we'll follow your suggestion to split PRs.